puppet/fabfile.py

```diff
@@ -128,6 +170,11 @@ def puppet_apply(dry_run='no'):
             " "+manifest_to_run+" "
             )
 
```

Comment:
Putting the upgrade here *DOES* mean that the upgrade happens automatically. I thought about it and came to the conclusion that everybody would just run them without thinking about it anyway (at least I would). If I typed `! deploy to server` and it came back with `migrations required`, I would instantly be like... "ok, `! deploy to server upgrade_db`" without thought.

I think it's ok to run them automatically for two reasons:
1. We always deploy to staging first, so that's where we'll catch most failures
2. We back up the db before running migrations